dist/angular-validation.js

```diff
@@ -665,6 +665,19 @@
         validMethod = (angular.isUndefined(validMethod)) ? $validationProvider.getValidMethod() : validMethod;
 
         /**
+         * A simple hack to provide global validate
+         */
+        if(!$validationProvider.validators){
+          $validationProvider.validators = [];
+          $validationProvider.validAll = function () {
+            return $validationProvider.validators.map(x=>x()).reduce((x,y)=>x&&y);
+          };
+        }
+        $validationProvider.validators.push(function () {
+          return checkValidation(scope, element, attrs, ctrl, validation, ctrl.$viewValue)
+        });
+
+        /**
```

Comment:
Well, i'll try to learn the grunt.
BTW, it struck me that maybe i can use the validate group to achieve my purpose. I'll try it later.